core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java

```diff
@@ -349,27 +349,34 @@ private void validateStackAuthentication(StackV4Request source) {
         return convertedSet;
     }
 
+    private boolean shouldCreateLoadBalancers(StackV4Request source) {
```

Comment:
I did something similar here in https://github.com/hortonworks/cloudbreak/pull/9675. It's actually probably going to be a bit of a pain to consolidate this change and that one because they both mess with this part of the code a lot.

Reply:
Yes, here: https://jira.cloudera.com/browse/CDPSDX-2728